Makefile

```diff
@@ -53,3 +59,30 @@ coursier:
 
 test: coursier output
 	JAVA_OPTS="-Xmx$(MEMORY)" ./coursier launch com.ggvaidya:shacli_2.12:0.1-SNAPSHOT -- validate shacl/omnicorp-shapes.ttl output/*.ttl
+
+
+# RoboCORD
+.PHONY: robocord-download robocord-output robocord-test
+robocord-download:
+	# rm -rf robocord-datas/${ROBOCORD_DATE}
+	-rm robocord-data
```

Comment:
What is this syntax (beginning dash)?

Reply:
It tells Make to keep going even if those commands fail. I've rewritten this code in 7bdc9df so that the robocord-data symlink is updated only if it doesn't exist OR is already a symlink, so that an existing robocord-data directory is left untouched.